net.kubux.Muehle.yml

```diff
@@ -0,0 +1,23 @@
+app-id: net.kubux.Muehle
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: muehle
+modules:
+  - name: muehle
+    buildsystem: simple
+    build-commands:
+      - tar -xjpvf muehle.tbz -C /
+    sources:
+      - type: file
+        url: http://www.kubux.net/muehle/muehle.tbz
+        dest-filename: muehle.tbz
+        sha256: f6d78747981113595145a326eded221afda857bfdd3cf92acaa90b313ed26300
+        size: 994654
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # store two numbers in muehle.rc: AI strength and boardsize in pixels
+  - --filesystem=xdg-config:create
```

Comment:
I don't think this is needed

Reply:
In order to make settings persistent, this is what I found works. I tried to be more specific with the file path but that did not create the file and the settings ended up not being persistent. Please advise.

Reply:
OK, thanks for the hint. Turns out that
  --filesystem=xdg-config:rw
does indeed suffice.

Reply:
> In order to make settings persistent, this is what I found works. I tried to be more specific with the file path but that did not create the file and the settings ended up not being persistent. Please advise.

by using XDG_CONFIG_HOME properly instead of `~/.config`. But I can't see the source code, so I'll just assume that's what's being done wrong. Also `--filesystem=xdg-config` give access to any config from any other app outside of the sandbox.